masonry_core/src/testing/helper_widgets.rs

```diff
@@ -734,12 +725,4 @@ impl<W: Widget> Widget for Recorder<W> {
     fn short_type_name(&self) -> &'static str {
         "Recorder"
     }
-
-    fn as_any(&self) -> &dyn std::any::Any {
-        self.child.as_any()
```

Comment:
This seems to be different semantics, right?

Reply:
True, it makes downcasting code "skip" the Recorder wrapper.

These downcasting tricks became mostly redundant since we removed `impl Widget for Box<dyn Widget>`, and they were a source of headaches before then.